packages/plugin-page-view-tracking-browser/src/page-view-tracking.ts

```diff
@@ -74,9 +74,8 @@ export const pageViewTrackingPlugin: CreatePageViewTrackingPlugin = (options: Op
         // eslint-disable-next-line @typescript-eslint/unbound-method
         globalScope.history.pushState = new Proxy(globalScope.history.pushState, {
           apply: (target, thisArg, [state, unused, url]) => {
+            target.apply(thisArg, [state, unused, url]);
```

Comment:
invoke `pushState` before `trackHistoryPageView`. this ensures that usages of `window.location` on page view tracking refers to the new page